mllib-local/src/test/java/dev/ludovic/netlib/blas/F2jBLASUtil.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.ludovic.netlib.blas;
+
+/**
+ * This is a workaround since class {@link F2jBLAS} are not public
+ */
+public class F2jBLASUtil {
```

Comment:
I don't think this matters at all, but I'd say 'final' here

Reply:
Can we use reflection to get `JavaBLAS` and move this class into a spark namespace package? Is this better than adding a `ludovic/netlib` package in Spark code?

Reply:
For just test purposes, the new package and shim seems fine, but I would be fine with either